stellarphot/utils/tests/test_magnitude_system_transforms.py

```diff
@@ -0,0 +1,174 @@
+# The goal here is to both test code and the data files
+from pathlib import Path
+
+import numpy as np
+import pytest
+from astropy.utils.data import get_pkg_data_path
+from pydantic import ValidationError
+
+from stellarphot.utils.magnitude_system_transforms import (
+    MagnitudeSystem,
+    MagnitudeSystemNames,
+    MagnitudeSystemTransform,
+    MagnitudeTransform,
+    PanStarrs1ToJohnsonCousins,
+)
+
+
+def _check_json_roundtrip(model):
+    """
+    Check that the model can be serialized to JSON and then deserialized
+    back to the same model.
+    """
+    json_str = model.model_dump_json()
+    new_model = model.model_validate_json(json_str)
+    assert model == new_model
+
+
+class TestMagnitudeSystem:
+    def test_can_create_and_jc_file_is_good(self):
+        jc_bands = ["U", "B", "V", "Rc", "Ic"]
+        sys = MagnitudeSystem(name=MagnitudeSystemNames.JC, passbands=jc_bands)
+        assert sys.name == MagnitudeSystemNames.JC
+        assert sys.passbands == jc_bands
+
+        # Check that Johnson-Cousins that we ship is as expected
+        from_data = Path(
+            get_pkg_data_path("data/JohnsonCousins.json", package="stellarphot.utils")
+        )
+        assert from_data.exists()
+        assert sys == MagnitudeSystem.model_validate_json(from_data.read_text())
+
+        # Check serialization roundtrip
+        _check_json_roundtrip(sys)
+
+    def test_no_unknown_systems(self):
+        with pytest.raises(ValidationError):
+            MagnitudeSystem(name="Unknown", passbands=["U", "B", "V", "R", "I"])
+
+    def test_panstarrs1_system_def(self):
+        ps1_file = Path(
+            get_pkg_data_path("data/pan-starrs1.json", package="stellarphot.utils")
+        )
```

Comment:
For consistency, do you want to check if `ps1_file` exists, since we ship that as well, no?